src/interpreter/mod.rs

```diff
@@ -306,7 +282,8 @@ impl<'a, 'tcx> EvalContext<'a, 'tcx> {
 
     pub fn monomorphize(&self, ty: Ty<'tcx>, substs: &'tcx Substs<'tcx>) -> Ty<'tcx> {
         let substituted = ty.subst(self.tcx, substs);
-        self.tcx.normalize_associated_type(&substituted)
+        let new = self.tcx.normalize_associated_type(&substituted);
+        new
```

Comment:
Oh, I think I was logging what monomorphizations where being performed for some reason, a month ago, and then I only deleted the logging, not the variable...